tests/system/apmserver.py

```diff
@@ -1,3 +1,8 @@
+from kibana import Kibana
+from es_helper import index_smap, index_span, index_error, apm_prefix
+from es_helper import cleanup, default_pipelines
+from helper import wait_until
+from beat.beat import INTEGRATION_TESTS, TestCase, TimeoutError
```

Comment:
Just noticed this. I think this is going to break the system tests, since it's before the `sys.path.append` on line 25?

Reply:
right, I actually didn't mean to check that in, was automatically done when running `make fmt`, but needs to be reverted for `make check-full` (haven't found time to look into the reason for that yet). Will revert.